packages/core/src/components/resize-sensor/resizeSensor.tsx

```diff
@@ -89,6 +90,7 @@ export class ResizeSensor extends AbstractPureComponent<ResizeSensorProps> {
     }
 
     public componentDidMount() {
+        this.observer = new ResizeObserver(entries => this.props.onResize?.(entries));
```

Comment:
this caused a runtime error in some test environments like jsdom where `ResizeObserver` is undefined. apparently components still get mounted in jsdom, whoops. I will fix the regression ASAP